src/ledger/LedgerTxn.cpp

```diff
@@ -3497,8 +3613,8 @@ LedgerTxnRoot::Impl::getOffersByAccountAndAsset(AccountID const& account,
             toPrefetch.emplace(trustlineKey(oe.sellerID, oe.selling));
         }
     }
-    prefetch(toPrefetch);
-
+    std::optional<LedgerKeyMeter> empty;
+    prefetchInternal(toPrefetch);
```

Comment:
DEX operations (which are the TX type that calls this function) are classic, so this should call `prefetchClassic` to get metered properly.